crates/ruff/src/flake8_to_ruff/isort.rs

```diff
@@ -1,10 +1,160 @@
+        ]
+        .map(std::string::ToString::to_string),
+    )
+}
+
+fn default_line_length() -> usize {
+    79
+}
+
+fn default_sections() -> Box<[String]> {
+    Box::new([
+        "FUTURE".to_string(),
+        "STDLIB".to_string(),
+        "THIRDPARTY".to_string(),
+        "FIRSTPARTY".to_string(),
+        "LOCALFOLDER".to_string(),
+    ])
+}
+
+fn default_known_future_library() -> HashSet<String> {
+    HashSet::from_iter(["__future__".to_string()])
+}
+
+fn default_multi_line_output() -> WrapMode {
+    WrapMode::Grid
+}
+
+fn default_ident() -> String {
+    "    ".to_string()
+}
+
+fn default_common_prefix() -> String {
+    "  #".to_string()
+}
+
 /// The [isort configuration](https://pycqa.github.io/isort/docs/configuration/config_files.html).
+// See https://github.com/PyCQA/isort/blob/3a72e069635a865a92b8a0273aa829f630cbcd6f/isort/settings.py
 #[derive(Debug, PartialEq, Eq, Serialize, Deserialize, Default)]
 pub struct Isort {
+    #[serde(default = "default_py_version")]
+    pub py_version: String,
+    #[serde(default)]
+    pub force_to_top: HashSet<String>,
+    #[serde(default = "default_skip")]
+    pub skip: HashSet<String>,
+    #[serde(default)]
+    pub extend_skip: HashSet<String>,
+    #[serde(default)]
+    pub skip_glob: HashSet<String>,
+    #[serde(default)]
+    pub skip_gitignore: bool,
+    #[serde(default = "default_line_length")]
+    pub line_length: usize,
+    #[serde(default)]
+    pub wrap_length: usize,
+    #[serde(default)]
+    pub line_ending: String,
+    #[serde(default = "default_sections")]
+    pub sections: Box<[String]>,
+    #[serde(default)]
+    pub no_sections: bool,
+    #[serde(default = "default_known_future_library")]
+    pub known_future_library: HashSet<String>,
+    #[serde(default)]
+    pub known_third_party: HashSet<String>,
+    #[serde(default)]
+    pub known_first_party: HashSet<String>,
+    #[serde(default)]
+    pub known_local_folder: HashSet<String>,
+    #[serde(default)]
+    pub known_standard_library: HashSet<String>,
+    #[serde(default)]
+    pub extra_standard_library: HashSet<String>,
+    #[serde(default)]
+    pub know_other: HashMap<String, HashSet<String>>,
+    #[serde(default = "default_multi_line_output")]
+    pub multi_line_output: WrapMode,
+    #[serde(default)]
+    pub forced_separate: Box<[String]>,
+    #[serde(default = "default_ident")]
+    pub ident: String,
+    #[serde(default = "default_common_prefix")]
+    pub common_prefix: String,
+    #[serde(default)]
+    pub length_sort: bool,
+    #[serde(default)]
+    pub length_sort_straight: bool,
+    #[serde(default)]
+    pub length_sort_sections: HashSet<String>,
+    #[serde(default)]
+    pub add_imports: HashSet<String>,
+    #[serde(default)]
+    pub remove_imports: HashSet<String>,
+    #[serde(default)]
+    pub append_only: bool,
+    #[serde(default)]
+    pub reverse_relative: bool,
+    #[serde(default)]
+    pub force_single_line: bool,
+    #[serde(default)]
```

Comment:
These may all need the `alias = "single-line-exclusions", alias = "single_line_exclusions"` treatment as seen in `src_paths`, to respect isort's acceptance of either casing style.

Reply:
I added such aliases too! Thanks! 🙏🏻